tools/configen/conf/configen.yaml

```
@@ -15,6 +15,11 @@ configen:
   # list of modules to generate configs for
   modules:
     - name: configen.samples.my_module
+      # Optional dict defining hydra flags for all classes in this module:
+      # default_flags:
+      #   _convert_ : 'all'
+      #   _recursive_ : True
```

```suggestion
      #   _convert_: all
      #   _recursive_: True
```

Since this is yaml, you do not need the quotes around strings.